formhero/forms/tests.py

```diff
@@ -1,4 +1,32 @@
-from django.test import TestCase
+from django.contrib.auth import get_user_model
+from formhero.apps.models import App
+from rest_framework.reverse import reverse
+from rest_framework.test import APITestCase
+from formhero.forms.views import CreateForm
+from rest_framework import status
 
 
-# Create your tests here.
+User = get_user_model()
+
+
+class RetrievePostFormTests(APITestCase):
+
+    def test_post_a_new_form(self):
+        app1 = App.objects.create(host_list=["testserver"])
+        handler = 'db'
+        data = {"app": app1.pk, "name": "Szymon", "handler": handler, "config": "test"}
```

Comment:
@szymonzmyslony Putting the dictionary values on separate lines makes the code more maintainable and readable.
https://dangerfarms.gitbooks.io/newcomers-guide/content/section-2/code_quality.html#python-specific-rules